site/content/Resources.html

```diff
@@ -5,16 +5,20 @@
 image: "images/bghome.jpg" 
 ---
 <html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="../../static/css/main.css" />
+    </head>
     <body>
-        <img src="/images/resources-header.jpg">
-<p><center><h2n>Interpretations</h2n></center></p>
+<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#ENGAGE"><font size="6" color=#51a2f3><b>ENGAGE</b></font></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#DOWNLOAD"><font size="6" color=#51a2f3 ><b>DOWNLOAD</b></font></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#LEARN"><font size="6" color=#51a2f3  ><b>LEARN</b></font></a></p>
```

Comment:
I think a table or nested divs using `flexbox` would be a better choice for aligning the link headers. Not sure what's best with Bootstrap3 (that is what the hamburg theme uses)